unittests/unittest_utils.py

```diff
@@ -173,5 +173,40 @@ def create_random_string(length=8):
     return "".join(random.choice(char_set) for x in range(length))
 
 
+def balanced_text(myText):
+    """
+    checks line by line that parens and quotations are balanced
+        @type myText: string
+        @param myText: input to be checked for balanced quotations and parens
+    """
```

Comment:
We use docstrings in google format:
https://cdcvs.fnal.gov/redmine/projects/glideinwms/wiki#Code-Documentation-Guidelines

PS Both VSCode and PyCharm allow to configure the docstring template to it

Reply:
Just got this email from Steve:

The glidein submitted ran and jobs matched to it, but it went held at the end of the job for reasons I have not yet figured out.
I don't think they are tied to the bug fix but would like to hold the ticket open until I figure that out.